qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManagePatientPage.java

```diff
@@ -24,4 +24,9 @@ public void searchPatientIdentifierOrPatientName(String text) {
         waitForTextToBePresentInElement(SEARCH_STATUS,text.substring(0, text.length() - 1));
     }
 
+    public void clickOnFirstPatient(String text) {
```

Comment:
Can you re-use this method from here https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/FindPatientPage.java#L29

Reply:
This method can only work for reference application not legacy ui, so if you are  working towards legacyui, please continue.

Reply:
Thanks did the needful